keps/sig-node/4381-dra-semantic-parameters/README.md

```diff
@@ -284,33 +280,37 @@ apiVersion: resource.k8s.io/v1alpha2
 spec:
   nodeName: worker-1
   driverName: cards.dra.example.com
-  partitionable:
+  namedResourcesWithAttributes:
     ...
 ```
 
-Here all resources associated with the `cards.dra.example.com` driver are to
-be placed under the `partitionable` field in the `NodeResourceSlice` object. As
-we add more "semantic models" in the future, *alternate* fields will be added
-at the same level as `partitionable`, and driver implementors will be able to
-choose which one they want to use to represent their resources.
+Such a model could be created to represent resources in a manner similar to the
+opaque strings passed over the tradition device plugin API to the kubelet. The
+one addition being that each named resource can have a set of arbitrary
+attributes attached to it.
+
+If a driver wanted to use a different semantic model to represent its resources,
+a new semantic model would need to be defined inside Kuberenetes, and a field
+would need to be added to this struct at the same level as
+`namedResourcesWithAttributes`. Driver implementors would then have the option
+to set this new field instead.
 
-If a new model is added to the schema but clients are not updated,
+**Note:** If a new model is added to the schema but clients are not updated,
 they'll encounter an object with no information from any known semantic model
-when they serialize into their known version of a NodeResourceSlice. This tells
-them that they cannot handle the object because the API was extended.
+when they serialize into their known version of a `NodeResourceSlice`. This
+tells them that they cannot handle the object because the API has been extended.
 
 Drivers can use different semantic models by publishing multiple
-NodeResourceSlice objects, as long as each model represents a distinct set of
+`NodeResourceSlice` objects, as long as each model represents a distinct set of
 resources. Whether the information about resources of one particular semantic
-model must fit into one NodeResourceSlice object or can be distributed across
-many depends on the semantic model and how it describes resources. The size of
-each object is a hard limit that must be taken into account when designing a
-semantic model and preparing NodeResourceSlice objects.
+model must fit into one NodeResourceSlice object (or distributed across many)
+depends on how that particular semantic model describes its resources. In all
+cases, the size of each object is a hard limit and one must take this into
+account when designing a semantic model and preparing NodeResourceSlice objects
+for it.
 
 Below is an example of a driver that provides two discrete GPU cards using the
-`partitionable` model. The first card can only be used as a whole. The second
-card can be used as a whole, split up into four equal pieces, in halves, or one
-half and two quarters:
+hypothetical `namedResourcesWithAttributes` model described above:
```

Comment:
I wouldn't say "hypothetical" because we will implement it. Maybe "prototype"? (non-blocking comment)

Reply:
removed